### PR TITLE
custom url added into lib. 

### PR DESCRIPTION
**Pull request by ricardmc**
Small modifications have been applied but I have to say sorry. All linting and indentation changes have been declared as code changes.

So, if you need a short meeting/call to check the modifications, I'll be totally available for that.

### CODE REVIEW

**Review by apenalver2 — CHANGES_REQUESTED** [review]
Compilating using ng build --prod the process is crashing.

Please, can you fix it?

Thank you

**Review by apenalver2 — APPROVED** [review]

File: projects/systelab-translate/README.md [hunk 1]
```diff
@@ -8,6 +8,19 @@ Library with I18N tools to speed up our Angular developments
 npm install systelab-translate --save
 ```
 
+## Importing the library
```
Comment (alfonsserra) [inline projects/systelab-translate/README.md:11]: This is part of How to use the library. It is a sub-case as it is not mandatory.

File: projects/systelab-translate/README.md [hunk 2]
```diff
@@ -8,6 +8,19 @@ Library with I18N tools to speed up our Angular developments
 npm install systelab-translate --save
 ```
 
+## Importing the library 
+
+These lines should be added in app.module file
+```javascript 
+  // required var declaration
+  let _translationFilesUrl = null
```
Comment (alfonsserra) [inline projects/systelab-translate/README.md:16]: Do not use _ prefix. for variables. see Angular Code Conventions.
Assign to something.

File: projects/systelab-translate/README.md [hunk 3]
```diff
@@ -78,6 +95,15 @@ This two files are mandatory if you are going to use the language.
 
 Sample files names are: MessageBundle_es.json, MessageBundle_en.json, MessageBundle_en_US.json or MessageBundle_pt_BR.json
 
+###### restAPI usage:
```
Comment (alfonsserra) [inline projects/systelab-translate/README.md:98]: It is not rest. It is direct http

File: projects/systelab-translate/README.md [hunk 4]
```diff
@@ -78,6 +95,15 @@ This two files are mandatory if you are going to use the language.
...
+
+In order to provide a json object to the library, the declared var _translationFilesUrl in app.module shall be filled with a valid restAPI url:
+
+```javascript
+  //store a valid restAPI url here
+  let _translationFilesUrl = './restAPI_URL_to_be_replaced';
```
Comment (alfonsserra) [inline projects/systelab-translate/README.md:104]: Do not use _
Use const

File: projects/systelab-translate/src/lib/systelab-translate.module.ts [hunk 5]
```diff
@@ -7,34 +7,51 @@ import { NumberFormatPipe } from './number-format.pipe';
 import { DecimalPipe, Location, LocationStrategy, PathLocationStrategy } from '@angular/common';
 
 // AoT requires an exported function for factories
-export function httpLoaderFactory(http: HttpClient, location: Location) {
-	return new LocalizableTranslateStaticLoader(http, location);
+export function httpLoaderFactory(http: HttpClient, location: Location, url: string) {
+  return new LocalizableTranslateStaticLoader(http, location, url);
 }
 
+export interface ISystelabTranslateModule {
```
Comment (alfonsserra) [inline projects/systelab-translate/src/lib/systelab-translate.module.ts:14]: Do not use I prefix. We do not have that code convention.

Rename the interface to something like TranslationConfiguration

Make the customUrl optional.

File: projects/systelab-translate/src/lib/systelab-translate.module.ts [hunk 6]
```diff
@@ -7,34 +7,51 @@ import { NumberFormatPipe } from './number-format.pipe';
...
+  customUrl: string;
+}
+
+export const _translationFilesUrl = null;
```
Comment (alfonsserra) [inline projects/systelab-translate/src/lib/systelab-translate.module.ts:18]: Do not use _ prefix.

**Review by alfonsserra — CHANGES_REQUESTED** [review]

**Comment by olgapuig** [comment]
@ricardmc Please, reject this pull-request in case it's no longer needed.

**Comment by ricardmc** [comment]
unnecessary change for the moment. I close it to allow angular 12 upgrade as Olga requested

**Pull request closed without merging**